go.mod

```diff
@@ -21,7 +21,9 @@ require (
 	github.com/hashicorp/go-msgpack v0.0.0-20150518234257-fa3f63826f7c // indirect
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/hashicorp/raft v1.0.0 // indirect
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
```

Comment:
please move it to separated gomod

Reply:
done b3ae9f252ccb6e90efcf9f72512765a9df71cff4

However, I think this will only work after merging this branch to `master`.

Might need some post-merge cleanup